Games/types/Mafia/roles/cards/MakeKillStronger.js

```diff
@@ -14,18 +14,18 @@ module.exports = class MakeKillStronger extends Card {
           return !this.data.hasUsedStrength;
         },
         action: {
-          labels: ["absolute"],
+          labels: ["strong"],
           priority: PRIORITY_MODIFY_ACTION_LABELS,
           run: function () {
             if (this.target === "No") return;
 
             for (let action of this.game.actions[0]) {
               if (
                 action.actors.includes(this.actor) &&
-                action.hasLabels(["kill"])
+                action.hasLabel("kill")
               ) {
                 action.power = Infinity;
-                action.labels.push("absolute");
+                action.labels = [...action.labels, "absolute", "strong"];
```

Comment:
Should it be using absolute here? If we're using strong, it shouldn't also have absolute, right?